### PR TITLE
Add conversion script from 0.8 to 1.0

### LINKED ISSUE #74: add conversion script from 0.8 to 1.0
Have a script that I can run on a folder with mdsynthesis `<=0.8` folders and convert them to the new datreant format automatically.

### PR DESCRIPTION
**Pull request by kaceyaurum**
Fix #74

### CODE REVIEW

File: src/scripts/converter.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+        old = json.load(fh)
+    mds.Sim(folder, categories=old['categories'], tags=old['tags'])
+
+if __name__ == '__main__':
+    import argparse
+    parser = argparse.ArgumentParser(description="Convert old sims to new sims.", formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+    parser.add_argument("directories", metavar="DIRECTORY", nargs="+", help="one or more directories that are being processed")
+    args = parser.parse_args()
+
+for dir in args.directories:
```
Comment (kain88-de) [inline src/scripts/converter.py]: these lines need also be indented. They should only execute if this is called from the command line

File: src/scripts/converter.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+import mdsynthesis as mds
+import json
+from os import path
+from glob import glob
+
+def convert(folder):
+    json_file = glob(path.join(folder, 'Sim*'))[0]
+    with open(json_file) as fh:
+        old = json.load(fh)
+    mds.Sim(folder, categories=old['categories'], tags=old['tags'])
```
Comment (kain88-de) [inline src/scripts/converter.py]: We also need to convert the atomgroups stored in the sim object. @dotsdl what else is stored in a  sim?

**Review by kain88-de — DISMISSED** [review]
Great thanks.

There are still some changes left we need to make to merge this.

**Comment by kain88-de** [comment]
@dotsdl and @richardjgowers  ding ding ding

I made changes that this should also add the universe definition and atom selections into the new format.  A review is appreciated so we can get the releases out.

@kaceyreidy I haven't tested this much could you give it a spin? Thanks

**Comment by richardjgowers** [comment]
How difficult would it be to add a test? Ie roll back mds installation, create Sim with tags and cats, tar ball it and use as test fixture?

**Comment by kaceyaurum** [comment]
@kain88-de I fixed some things. (I didn't have any atomselections, so I made that a try statement.) I don't know how to make a test, but I tried it on a sim, and it worked.

**Comment by kain88-de** [comment]
Thanks @kaceyreidy 

You can have a look here: https://github.com/datreant/datreant/pull/160

It's the same PR but for the datreant. In it I added tests. I will have some time next week to have a look at this again.

**Comment by kain88-de** [comment]
ping @dotsdl @richardjgowers 

The script is installable now. There are tests and the basic build-up of the script is similar to the datreant version.

**Comment by dotsdl** [comment]
Thanks all for this work. Merging!

**Pull request merged**